packages/frontend/lib/model/extract-capi.ts

```diff
@@ -232,6 +232,11 @@ export const extract = (data: {}): CAPIType => {
                 : undefined,
             email: 'none',
         },
+        mainMediaElements: getArray<ImageBlockElement>(
```

Comment:
Can the main element be a non-image? I.e. video etc.?

Reply:
Good point. I'll change this to an array of `CAPIElement`s for now, until we determine exactly which types they could take.